frontend/src/components/broadcast/broadcast_second.js

```diff
@@ -0,0 +1,64 @@
+import React, { Component } from "react";
+import OwlCarousel from "react-owl-carousel";
+import "owl.carousel/dist/assets/owl.carousel.css";
+import "owl.carousel/dist/assets/owl.theme.default.css";
+import "./owl.css";
+import Card from './card.js'
+
+
+
+
+export class Owldemo1 extends Component {
+    render() {
+
+
+    return (
+
+        <div className="Bottom_bgbox">
+        <div className="Bottom_bgbox_first">
+      <div className="motive">
+          <h1 style={{ color: "#1B2431" }}>Previsely Broadcast</h1>
```

Comment:
Check the spelling error.

Reply:
but in UI design, it was given completely white when the popout open

Reply:
Yes, I'm aware. Giving an opacity will make it look better. 😃 Anything that enhances the UI is appreciated.

Reply:
ok